### PR TITLE
rsx-debug: Dump windows origin/pixel center.

### PR DESCRIPTION
**Pull request by vlj**

### CODE REVIEW

File: rpcs3/Emu/RSX/GCM.cpp [hunk 1]
```diff
@@ -763,6 +763,33 @@ rsx::primitive_type rsx::to_primitive_type(u8 in)
 }
 
 
+#define CELL_GCM_WINDOW_ORIGIN_TOP  0
```
Comment (tambry) [inline rpcs3/Emu/RSX/GCM.cpp]: Defines should be avoided according to our style guide. Why remove them from the enum?

**Comment by vlj** [comment]
Use enum instead of define.

File: rpcs3/Emu/RSX/GCM.cpp [hunk 2]
```diff
@@ -762,6 +762,34 @@ rsx::primitive_type rsx::to_primitive_type(u8 in)
 	throw new EXCEPTION("Unknow primitive type %d", in);
 }
 
+enum
+{
+	CELL_GCM_WINDOW_ORIGIN_TOP = 0,
```
Comment (tambry) [inline rpcs3/Emu/RSX/GCM.cpp]: Better, but why move them from the header? I don't think there's any reason to split up the CELL_GCM enum.

Comment (vlj) [inline rpcs3/Emu/RSX/GCM.cpp]: I prefer that the non class enum are not available outside of GCM.cpp so that all code uses the enum class ones.

**Pull request merged**